packages/core/src/diff/changes/field.ts

```diff
@@ -218,6 +221,7 @@ export function fieldDeprecationRemovedFromMeta(args: FieldDeprecationRemovedCha
   return {
     type: ChangeType.FieldDeprecationRemoved,
     criticality: {
+      // @todo: Add a reason for why is this dangerous... Why is it??
```

Comment:
^ very curious why... It doesn't change behavior/how clients interact. "Dangerous" seems extreme.

Reply:
Not sure to be honest. My initial take is that it shouldn't be dangerous as you just remove a metadata from a field that's already exists (and can be queries), but maybe @kamilkisiela has a different take